content/concepts/mcp/control-plane-connector.md

```diff
@@ -14,16 +14,29 @@ aliases:
 
 ### Argo CD
 
-{{< hint "important" >}}
-Argo CD is currently unable to target a managed control plane as an external cluster.  Argo CD must be able to create a service account on the target cluster, and this is a permission Upbound disallows on managed control planes. Upbound is working to fix this gap; the documentation in this article will be updated once it has been fixed.
-{{< /hint >}}
+#### Configure Argo CD
+
+To configure Argo CD for Annotation resource tracking, edit the argocd-cm
```

Comment:
Is `Annotation` supposed to be capitalized? 

Is `argocd-cm` a specific file?

Reply:
used it from this page: https://docs.crossplane.io/knowledge-base/integrations/argo-cd-crossplane/